src/status_line_manager.ts

```diff
@@ -19,9 +20,9 @@ export class StatusLineManager implements Disposable {
     private _cmdText = "";
     private _msgText = "";
 
-    private statusBar: StatusBarItem;
+    private _statusline = "";
```

Comment:
nit: we don't have a getter for this, so why use the underscore prefix?

Reply:
Just something left from before.